cmd/registry_push.go

```diff
@@ -127,7 +127,7 @@ func (o *pushOptions) RunPush(ctx context.Context, args []string) error {
 
 	client := authn.NewClient(cred)
 
-	pusher := ocipusher.NewPusher(client, newPushProgressTracker(o.Printer))
+	pusher := ocipusher.NewPusher(client, false, newPushProgressTracker(o.Printer))
```

Comment:
Let's remember to have a flag for PlainHTTP, but I agree that this is not the purpose of this PR